be/src/runtime/stream_load/stream_load_executor.cpp

```diff
@@ -251,6 +251,11 @@ Status commit_txn_internal(const TLoadTxnCommitRequest& request, int32_t rpc_tim
         bool visible =
                 wait_txn_visible_until(ctx->auth, request.db, request.tbl, request.txnId, ctx->load_deadline_sec);
         return visible ? Status::OK() : status;
+    } else if (result.__isset.allow_retry && result.allow_retry) {
+        LOG(WARNING) << "commit transaction " << request.txnId << " failed, will retry after sleeping "
+                     << result.retry_interval_ms << "ms. errmsg=" << status.get_error_msg();
+        std::this_thread::sleep_for(std::chrono::milliseconds(result.retry_interval_ms));
```

Comment:
not sure which thread this executor will be on, will it exhaust the threads in thread pool if several `txn_commit`s are all in sleep status.

Reply:
> will it exhaust the threads in thread pool...
Yes, it's possible, but this behavior is the same as waiting for data loading finished, i.e, sleep here for 5 seconds is just like the data loading takes extra 5 seconds.